### PR TITLE
Develop merge to main

### PR DESCRIPTION
**Pull request by dinakar-m**

### CODE REVIEW

File: {{cookiecutter.app_name}}/{{cookiecutter.app_name}}/Common/RxSwiftHelper.swift [hunk 1]
```diff
@@ -0,0 +1,30 @@
+//
+//  RxSwiftHelper.swift
```
Comment (rokon-mlbd) [inline {{cookiecutter.app_name}}/{{cookiecutter.app_name}}/Common/RxSwiftHelper.swift:2]: File name `RxSwiftHelper` is not appropriate

Comment (dinakar-m) [reply {{cookiecutter.app_name}}/{{cookiecutter.app_name}}/Common/RxSwiftHelper.swift]: @rokon-mlbd could you check again?
below name is good? if not please suggest what name should be fine.
RsSwiftGenerator.swift

Comment (rokon-mlbd) [reply {{cookiecutter.app_name}}/{{cookiecutter.app_name}}/Common/RxSwiftHelper.swift]: @dinakar-m 
Naming convension for extension I and many others follow is `R+Extension`

Comment (dinakar-m) [reply {{cookiecutter.app_name}}/{{cookiecutter.app_name}}/Common/RxSwiftHelper.swift]: @rokon-mlbd done, https://github.com/monstar-lab-oss/swiftUI-template/pull/10/commits/394c8c25a7bd39162a919346cd26f713992c4cfb

**Pull request merged**